components/automate-ui/src/app/components/create-object-modal/create-object-modal.component.ts

```diff
@@ -94,6 +97,11 @@ export class CreateObjectModalComponent implements OnInit, OnDestroy, OnChanges
     this.createForm.controls.addPolicies.setValue(this.addPolicies);
   }
 
+  updateTeamCheckbox(event: boolean): void {
```

Comment:
[nit] `event` seems a bit oddly named, perhaps? If it's a boolean, I'd go with `checked`. But I might be missing something.

Reply:
I agree, once again following the policies pattern. I'll update em both.